src/main/java/me/yukun99/ip/core/Ui.java

```diff
@@ -0,0 +1,272 @@
+package me.yukun99.ip.core;
+
+import me.yukun99.ip.tasks.Task;
+
+import java.io.IOException;
+import java.util.Objects;
+
+/**
+ * Class used to handle message the Ui and responses from the HelpBot.
+ */
+public class Ui {
+	// Name of the HelpBot.
+	private final String name;
+	// List of tasks to be completed.
+	private final TaskList taskList;
+	// Storage instance from the current HelpBot.
+	private final Storage storage;
+	// TaskFinder instance from the current HelpBot.
+	private final TaskFinder taskFinder;
+
+	// Field placeholders.
+	private static final String NAME_PLACEHOLDER = "%name%";
+	private static final String TASKNUM_PLACEHOLDER = "%tasks%";
+
+	// Reply header/footer.
+	private static final String REPLY_HEADER = "===================To Your Royal Highness===================";
+	private static final String REPLY_FOOTER = "============================================================";
+
+	// Line separator.
+	private static final String NEW_LINE = System.lineSeparator();
+
+	// Command help prefix.
+	private static final String CMD_PREFIX = "  > ";
+
+	// Command usage descriptions.
+	private static final String HELP_LIST = "view all your tasks, optionally (troublesome for me) by date";
```

Comment:
I like the specific help commands, makes it more user-friendly 👍

Reply:
Thanks!